jib-core/src/main/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStep.java

```diff
@@ -95,8 +95,8 @@ public ImagesAndRegistryClient call()
           LayerCountMismatchException, BadContainerConfigurationFormatException,
           CacheCorruptedException, CredentialRetrievalException {
     EventHandlers eventHandlers = buildContext.getEventHandlers();
-    try (ProgressEventDispatcher progressEventDispatcher =
-            progressEventDispatcherFactory.create("pulling base image manifest", 2);
+    try (ProgressEventDispatcher progressDispatcher =
+            progressDispatcherFactory.create("pulling base image manifest", 3);
```

Comment:
`progressDispatcher.newChildProducer()` (which decrements one progress unit) can be called up to 3 times inside this method.